scripts/linkbot/monitor-raspbian-os.js

```diff
@@ -35,13 +35,13 @@ function getNewRaspbian(target, url, separator) {
         var document = dom.window.document;
         var refs = document.getElementsByTagName("a");
         var test = Array.from(refs)
-            .filter((ref) => ref.textContent.match("raspios-.*-lite.*.zip$"))
+            .filter((ref) => ref.textContent.match("raspios-.*-lite.*.(zip|xz)$"))
             .reduce((acc, element) => {
-                acc.push(element.textContent.match("raspios-.*-lite.*.zip$").input);
+                acc.push(element.textContent.match("raspios-.*-lite.*.(zip|xz)$").input);
```

Comment:
would it be possible to check for the `img` extension too and maybe exclude the files we're sure not to want (maybe `.md5` or something...)?